docs/core/extensions/generic-host.md

```diff
@@ -26,7 +26,7 @@ The main reason for including all of the app's interdependent resources in one o
 
 The host is typically configured, built, and run by code in the `Program` class. The `Main` method:
```

Comment:
This is not necessarily something that needs to be corrected, but the reference to the Main method is out of date, as the snippet shows top level statements.

Reply:
I'll update this as part of a related issue I'm tracking [35897](https://github.com/dotnet/docs/issues/35897).